pkg/clients/rds/rds.go

```diff
@@ -199,6 +201,55 @@ func GenerateRestoreDBInstanceInput(name, password string, p *v1beta1.RDSInstanc
 	return c
 }
 
+// GenerateRestoreDBInstanceFromSnapshotInput from RDSInstanceSpec
+func GenerateRestoreDBInstanceFromSnapshotInput(name, password string, p *v1beta1.RDSInstanceParameters) *rds.RestoreDBInstanceFromDBSnapshotInput {
```

Comment:
As it stands, the provider will never set the master password for instances restored from snapshots: We don't use `password` here because the underlying `RestoreDBInstanceFromDBSnapshot` AWS API call doesn't support it, and `GenerateModifyDBInstanceInput` disregards password changes. What's the best approach here? Should we just document this limitation or attempt to set the master password after the database is restored?

Reply:
But the existing password works I assume. Then I'm not sure if this should be considered an issue specific to restoring, although I guess it's more likely that the user want a password change in this case.

I'm fine with documenting this via an issue, and then we can follow up that issue when there's interest and time. Hopefully we can find a general solution, instead of a workaround here.

Reply:
Agreed. Documented in #1121.